internal/proto/stdout_stderr_writer.go

```diff
@@ -0,0 +1,133 @@
+package proto
```

Comment:
it seems really weird for all this logic to live in the protobuf package...it doesn't really have anything to do with protos?

Reply:
I worried a bit about that and then covered my eyes and rationalized it by "well, it's operating on `Output`s". I think that after the change you suggested, this is little enough code, and sufficiently `Output`-specific, it might actually be ok to leave in this package?

Reply:
:+1: